VideoLocker/res/values/colors.xml

```diff
@@ -35,6 +35,7 @@
     <color name="pink_3">#b72667</color>
     <color name="pink_2">#e86caa</color>
     <color name="pink_1">#ed9bc7</color>
+    <color name="pink_0">#F8E0E6</color>

```

Comment:
@aleffert  created a new task 
https://openedx.atlassian.net/browse/MA-766  
we will make all the changes at once?